packages/amplify-graphql-api-construct-tests/src/__tests__/relationships/composite-pk/graphql/schema-primary-cpk-4sk.graphql

```diff
@@ -0,0 +1,9 @@
+type PrimaryCPKSKFour @model {
+  id: ID! @primaryKey(sortKeyFields: ["skOne", "skTwo", "skThree", "skFour"])
+  skOne: ID!
+  skTwo: ID!
+  skThree: ID!
+  skFour: ID!
```

Comment:
Are we likely to see a difference from 3-4 keys? I know we have behavior breakpoints from 0-1 and 1-2, but beyond that are we testing anything meaningful?